### PR TITLE
Resolve minor eslint warnings

### PR DESCRIPTION
**Pull request by fiskus**

### CODE REVIEW

File: catalog/app/utils/AWS/Signer.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import SignerV4 from 'aws-sdk/lib/signers/v4'
 import * as React from 'react'
 import * as redux from 'react-redux'
 
-import * as Auth from 'containers/Auth'
+import * as authSelectors from 'containers/Auth/selectors'
```
Comment (nl0) [inline catalog/app/utils/AWS/Signer.js:5]: what's the reason for this change?

**Review by nl0 — APPROVED** [review]

File: catalog/app/utils/AWS/Signer.js (hunk 1 above)
Comment (fiskus) [reply catalog/app/utils/AWS/Signer.js:5]: Cycle dependency:
Layout -> NavBar -> CatalogSettings -> AWS -> Signer,Credentials etc -> Auth -> SignIn, SIgnOut etc -> Layout

Comment (nl0) [reply catalog/app/utils/AWS/Signer.js:5]: oh i see, thanks for catching this. but how was it working before?

**Review by nl0 — APPROVED** [review]

File: catalog/app/utils/AWS/Signer.js (hunk 1 above)
Comment (fiskus) [reply catalog/app/utils/AWS/Signer.js:5]: ES6 allows cycle dependencies. So, it's a working code, but it's dangerous and has lint warning
